README.md

```diff
@@ -35,7 +35,7 @@ To offload computations to the GPU the following is also required
 * OpenCL1.1+
 * pyopencl
 * clFFT
```

Comment:
If I make Docker image without clFFT library it also works.

```suggestion
```

Also `Steps for running on GPU` details the clfft can be removed.